lib/hardware/modules/div/iob_div_subshift_frac/iob_div_subshift_frac.py

```diff
@@ -13,22 +22,181 @@ def setup(py_params_dict):
                 },
                 "descr": "Clock, clock enable and reset",
             },
-            # TODO: Remaining ports
+            {
+                "name": "dividend_i",
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "dividend",
+                        "width": "DATA_W",
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "divisor_i",
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "divisor",
+                        "width": "DATA_W",
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "status",
+                "descr": "",
+                "signals": [
+                    {
+                        "name": "start",
+                        "direction": "input",
+                        "width": 1,
+                        "descr": "Start signal",
+                    },
+                    {
+                        "name": "done",
+                        "direction": "output",
+                        "width": 1,
+                        "descr": "Done signal",
+                    },
+                ],
+            },
+            {
+                "name": "quotient_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "quotient",
```

Comment:
The quotient port is not being driven.
In the original module, it had an assign driving it:
https://github.com/IObundle/iob-soc/blob/8f10f1bd5fd583e04d524879af6ef14e0211bb2f/lib/hardware/modules/div/iob_div_subshift_frac/hardware/src/iob_div_subshift_frac.v#L32

We need to add a snippet with this assign.

I'm not sure how the test is passing with a floating quotient port.